components/sysutils/gnupg/gnupg.p5m

```diff
@@ -125,6 +126,7 @@ file path=usr/share/locale/da/LC_MESSAGES/gnupg2.mo
 file path=usr/share/locale/de/LC_MESSAGES/gnupg2.mo
 file path=usr/share/locale/el/LC_MESSAGES/gnupg2.mo
 file path=usr/share/locale/en/LC_MESSAGES/gnupg2.mo
+#file path=usr/share/locale/en/en@boldquot/LC_MESSAGES/gnupg2.mo
 file path=usr/share/locale/en@boldquot/LC_MESSAGES/gnupg2.mo
```

Comment:
Something must be wrong on your build machine. When I generate a new sample-manifest this line is missing.

Reply:
Maybe you started from a polluted build folder?
Try to run
gmake clean && gmake sample-manifest
and see whether the generated sample-manifest is correct.

Reply:
Yes, that must've been the case. When I run it properly, the line disappears. I removed it from both files. Thanks.